pkgs/development/libraries/libnut/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchgit }:
```

Usually people put a space here, or even better:

```suggestion
{ stdenv
, lib
, fetchgit
}:
```

---

Done! Thank you for the suggestion.